custom_components/mbapi2020/sensor.py

```diff
@@ -116,6 +116,9 @@ def state(self):
         if self._internal_name == "lastParkEvent":
             if self._state:
                 return datetime.fromtimestamp(int(self._state))
+        elif self._internal_name == "chargingpowerkw":
+            if self._state:
+                return round(float(self._state), 1)
```

Comment:
Thanks for this PR, is on my list since years :-) _state could be non float with an unknown value like None, "error" - this should be checked.

Reply:
Okay I've did a check to see if it as float or int.